santosfamilyfoundation/RuinaMain
# [RUIN-207] Fix duplicate auto-save files

# Description

This change updates the auto-save code so that duplicate saved states for the same report are not shown in the file picker when the user selects an in-progress report to reload. I will eventually be updating the auto-save code to save reports with a better naming scheme, but this is one step to increase the usability of the multiple in-progress reports feature!

# Testing
1. Create a report.
2. Click random options in the quick survey.
3. Close the app.
4. Reopen the app and click continue report. Confirm that there are no duplicates.
5. Open the most recent report (should be at the top).
6. Confirm that the report re-loads successfully.
